src/web/client/dal/remoteFetchProvider.ts

```diff
@@ -298,6 +324,11 @@ async function createContentFiles(
     }
 }
 
+function getRootPageId(contentValue: string) : string {
```

Comment:
You do not need this function. Use GetFileContent() in commonUtil file. You can use getAttributePath() function to convert V1 and V2 field values to this format and then GetFileContent will do the rest for you. You dont need to do V1 and V2 specific code handling in this fetchProvider at all.